src/sentry/static/sentry/app/components/events/interfaces/utils.jsx

```diff
@@ -5,7 +5,7 @@ import queryString from 'query-string';
 import {defined} from 'app/utils';
 
 export function escapeQuotes(v) {
-  return v.replace(/"/g, '\\"');
+  return v.replace(/\\/g, '\\\\').replace(/"/g, '\\"');
```

Comment:
Hmm maybe we should move this to app/utils for https://github.com/getsentry/sentry/pull/11867